tests/e2e/cucumber/features/smoke/link.feature

```diff
@@ -103,11 +103,11 @@ Feature: link
       | resource  |
       | lorem.txt |
     When "Brian" opens the public link "textLink"
-    Then for "Brian" file "shareToBrian.txt" should be selected
+    And "Brian" closes the file viewer
```

Comment:
Hm, I get that this is somehow an implicit verification step of `being in a file viewer`... but reads super weird. What do you think about introducing a generic step for `Then "Brian" is in a file viewer` or the like? @ScharfViktor any hints here?

Reply:
You are right, the `open public link` step does not contain any checking, and `close file viewer` just clicks the close button. 
need implement step that we can ensure that text editor was opened. A step should be implemented to ensure that the text editor has been opened.

added to my TODO list

Reply:
Not necessarily the text editor. I think for a first step it's already sufficient to have a check for any editor / viewer. That's already fulfilled if the close button in the top bar is present. So with the implementation that @JammingBen chose the tests are doing what they should... but they are hard to understand from just reading the Gherkin.